app/features/conference/components/Conference.js

```diff
@@ -81,7 +85,8 @@ class Conference extends Component<Props, *> {
     componentDidMount() {
         const parentNode = this._ref.current;
         const room = this.props.match.params.room;
-        const domain = this.props.match.params.domain || config.defaultDomain;
+        const domain = this.props.match.params.domain
+            || this.props._defaultDomain;
```

Comment:
This needs to also consider `config.defaultDomain` in case the user entered nothing.

Reply:
The reducer actually sets that as default value for defaultDomain. 
You must be experiencing this because of redux-persist. 

you'll need to flush previous values.

They way I do it is.
Inspect element -> React Tab -> Expand Root and Provider -> Select PeristorGate -> in right side you will see props -> Expand persistor -> You will see functions -> right click on flush function. and select "execute function"

Reply:
That will only work the first time. Go ahead and edit the field in settings and set it empty.

Reply:
@saghul 
I guess this can be fixed with valid server checking since "blank" will not be a valid server. Since we currently don't have error display in Conference component, this could be handled by that also.
But keeping the current functionality in mind, I think this is good to go for this PR. WDYT?

Reply:
If the user enters a URL and deletes it later it must continue to work. I’ll test again later today 👍

Reply:
> If the user enters a URL and deletes it later it must continue to work.

I guess then, I should add something like this in middleware. 
If its blank, then `dispatch(setDefaultServerURL(config.defaultServerURL)`.
WDYT?

Reply:
Okay. Then I'll set it `undefined`.